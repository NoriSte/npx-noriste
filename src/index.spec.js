"use strict";

const CaptureStdout = require("../vendor/capture-stdout");
const links = require("./links");

describe("npx noriste", () => {
  test("Should contain all the social links", async () => {
    const captureStdout = new CaptureStdout();
    captureStdout.startCapture();

    require("./index");

    captureStdout.stopCapture();
    const output = captureStdout.getCapturedText().join(",");
    captureStdout.clearCaptureText();

    links.forEach(([key, link]) => expect(output).toEqual(expect.stringContaining(link)));
  });
});
