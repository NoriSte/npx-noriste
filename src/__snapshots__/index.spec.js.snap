// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`npx noriste Should print all the info 1`] = `
"[38;2;255;51;102m[39m
[38;2;255;51;102m   â•­â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â•®[39m
   [38;2;255;51;102mâ”‚[39m                                                                    [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [38;2;255;51;102m[1mStefano Magni (NoriSte)[22m[39m                                          [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   Passionate, positive-minded Front-end developer.                 [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   JavaScript and testing lover.                                    [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m                                                                    [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m      Twitter: [22m[38;2;255;51;102mhttps://twitter.com/NoriSte[39m                       [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m     LinkedIn: [22m[38;2;255;51;102mhttps://www.linkedin.com/in/noriste/[39m              [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m       GitHub: [22m[38;2;255;51;102mhttps://github.com/NoriSte[39m                        [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m       Medium: [22m[38;2;255;51;102mhttps://medium.com/@NoriSte[39m                       [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1mStackOverflow: [22m[38;2;255;51;102mhttps://stackoverflow.com/users/700707/noriste[39m    [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m   Slides.com: [22m[38;2;255;51;102mhttps://slides.com/noriste[39m                        [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m       Resume: [22m[38;2;255;51;102mhttps://www.visualcv.com/stefano-magni/[39m           [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m          NPM: [22m[38;2;255;51;102mhttps://www.npmjs.com/settings/noriste/packages[39m   [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m   [1m       Dev.to: [22m[38;2;255;51;102mhttps://dev.to/noriste[39m                            [38;2;255;51;102mâ”‚[39m
   [38;2;255;51;102mâ”‚[39m                                                                    [38;2;255;51;102mâ”‚[39m
[38;2;255;51;102m   â•°â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â•¯[39m
[38;2;255;51;102m[39m
"
`;
